core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java

```diff
@@ -103,7 +103,7 @@ public LocalCacheManager(FileSystemContext fsContext) {
    * @return the corresponding page lock
    */
   private ReadWriteLock getPageLock(PageId pageId) {
-    return mPageLocks[(int) (pageId.getFileId() + pageId.getPageIndex()) % LOCK_SIZE];
+    return mPageLocks[Math.floorMod((int) (pageId.getFileId() + pageId.getPageIndex()), LOCK_SIZE)];
```

Comment:
will `Math.floorMod` handle overflow if `pageId.getFileId() + pageId.getPageIndex()` is larger than `LONG.MAX_VALUE`?

Reply:
We don't need it to handle - if the sum is overflow it will become negative, but the floorMod will always return non-negative when the second argument is positive.